homeassistant/components/shelly/binary_sensor.py

```diff
@@ -143,19 +143,6 @@ class RestBinarySensorDescription(RestEntityDescription, BinarySensorEntityDescr
         entity_registry_enabled_default=False,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
-    "fwupdate": RestBinarySensorDescription(
-        key="fwupdate",
-        name="Firmware Update",
-        device_class=BinarySensorDeviceClass.UPDATE,
-        value=lambda status, _: status["update"]["has_update"],
-        entity_registry_enabled_default=False,
```

Comment:
We missed that this was false and now changed to True in the update entity.
This means we now pull all devices every 60 seconds just to check if there is an update

Reply:
Can't we reduce the polling time to something like 1 hour or so?
Are there other entities that use polling?

Reply:
If we decrees it twice a day is more than enough (device itself check for updates every 12 hours so no reason to do it faster)
There is one binary sensor 
https://github.com/starkillerOG/home-assistant/blob/30fbe555550bb95411ba42348ab1e47607eeb71e/homeassistant/components/shelly/binary_sensor.py#L137
And two sensors:
https://github.com/starkillerOG/home-assistant/blob/30fbe555550bb95411ba42348ab1e47607eeb71e/homeassistant/components/shelly/sensor.py#L297

Which uses the same interval, we can't reduce the polling rate for them to a very low rate, they are diagnostic entities and enabled by users to debug problems.

Reply:
@thecode what would you suggest?
Make a new ShellyDeviceRestWrapper=UpdateCoordinator that polls with an intervall of 12 hours?
(I am afraid that would result in quite a bit of duplicate code...)

Reply:
What will happen if we disable the entities by default for now? Will the user be able to enable them? So we can address it in the next release? I am afraid adding another wrapper or doing major changes is too dangerous to do during beta.